indico/htdocs/js/indico/modules/registration/form/field.js

```diff
@@ -869,7 +869,7 @@ ndRegForm.directive('ndAccommodationField', function(url) {
                 updateAccommodationPostData('departureDate', moment(newValue).format('YYYY-MM-DD'));
             });
 
-            scope.$watch('accommodation.typeId', function(newValue) {
+            scope.$watch('accommodation.choice', function(newValue) {
```

Comment:
Ah, I somehow thought it's `userdata`... anyway, we also do things with `userdata.accommodation`..